Builder/src/KP2/HouseDirector.java

```diff
@@ -0,0 +1,25 @@
+package KP2;
+
+public class HouseDirector {
+	
+	private BuildingBuilder bBuilder = null;

```

Comment:
[Low] By default, all objects are initialized in *null* so you don't need to specify it.